### PR TITLE
Add documentation for alterCustomFieldDisplayValue  Hook

### PR DESCRIPTION
**Pull request by pradpnayak**

### CODE REVIEW

**Comment by pradpnayak** [comment]
For https://github.com/civicrm/civicrm-core/pull/16921

File: mkdocs.yml [hunk 1]
```diff
@@ -124,6 +124,7 @@ pages:
     - hook_civicrm_validateForm: hooks/hook_civicrm_validateForm.md
   - GUI Hooks:
     - hook_civicrm_activeTheme: hooks/hook_civicrm_activeTheme.md
+    - hook_civicrm_alterEntityRefParams: hooks/hook_civicrm_alterCustomFieldDisplayValue.md
```
Comment (MikeyMJCO) [inline mkdocs.yml]:
```suggestion
    - hook_civicrm_alterCustomFieldDisplayValue: hooks/hook_civicrm_alterCustomFieldDisplayValue.md
```

Comment (pradpnayak) [reply mkdocs.yml]: My bad, updated.

**Comment by MikeyMJCO** [comment]
Thanks @pradpnayak :-) merged!

**Comment by MikeyMJCO** [comment]
Hmm... Guess I should have waited and merged when the hook releases - got carried away!

**Pull request merged**